program/convert-weeks-to-hours/convert-weeks-to-hours.jl

```diff
@@ -0,0 +1,14 @@
+result = 0
+  
+# Prompt to enter
+total_hrs = 0
+
+println("Enter number of weeks : ")
+
+num_of_weeks = readline()
+num_of_weeks = parse(Int64, num) 
```

Comment:
`num` is not defined.

![image](https://user-images.githubusercontent.com/35031228/194118235-9fd9907c-0adb-4e0c-834b-d8dc3a219322.png)